### PR TITLE
Refactor - Advanced Heading.

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
Trello task - https://trello.com/c/mSbMJ7X5/144-uag-block-wise-cleanup

### Types of changes
Breaking change

### Checklist:
- [x] My code is tested
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by RajkiranBagal — APPROVED** [review]

**Comment by RajkiranBagal** [comment]
Let's keep PR titles consistent like: 

`refactor: Block Name`

File: src/blocks/advanced-heading/render.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+	return (
+		<div
+			className={ classnames(
+				className,
+				`uagb-editor-preview-mode-${ deviceType.toLowerCase() }`,
+				`uagb-block-${ block_id }`
+			) }
+		>
+			{ heading_html }
```
Comment (rohitp-bsf) [inline src/blocks/advanced-heading/render.js:63]: Maintain camel casing for variables and functions.

**Review by rohitp-bsf — CHANGES_REQUESTED** [review]

File: src/blocks/advanced-heading/settings.js [hunk 2]
```diff
@@ -0,0 +1,524 @@
+import React, { lazy, Suspense } from 'react';
+import lazyLoader from '@Controls/lazy-loader';
+
+const TypographyControl = lazy( () =>
+	import(
+		/* webpackChunkName: "chunks/advanced-heading/typography-control" */ '../../components/typography'
```
Comment (sandeshjangam) [inline src/blocks/advanced-heading/settings.js:6]: Creat an alise for components like @Components

**Review by sandeshjangam — CHANGES_REQUESTED** [review]
Need changes

File: src/blocks/advanced-heading/settings.js (hunk 2 above)
Comment (rohitp-bsf) [reply src/blocks/advanced-heading/settings.js:6]: Please pull `refactor-uag`

File: src/blocks/advanced-heading/render.js (hunk 1 above)
Comment (supriyajjadhav) [reply src/blocks/advanced-heading/render.js:63]: Updated all the variable names and function names

File: src/blocks/advanced-heading/settings.js (hunk 2 above)
Comment (supriyajjadhav) [reply src/blocks/advanced-heading/settings.js]: Updated in all the files

**Review by rohitp-bsf — APPROVED** [review]

**Pull request merged**